### ISSUE TITLE
show actual amount of pixels on screen of selected objects somewhere

### ISSUE THREAD

**Issue by mikestonecodes** [PhaserEditor2D/PhaserEditor2D-v3#76]
I have certain images that can be configured on a backend that feed into phaser, this works by replacing what file gets loaded into the cache before the game starts ( during preload step ) . I made a user component called autoResize, but I would like a way to copy the width and height *after the transformation* . so for example in the image has scale set to 0.2 0.2.  The original image is 223 x 223 , so I should be able to see somewhere this is 44 x 44 ( 223 x 0.2 ) . this way I can copy 44 x 44 into my component to make sure the image is correctly sized if another image is loaded that is not 223 x 223
![image](https://user-images.githubusercontent.com/1703689/92733492-04c70b80-f378-11ea-9cef-6da2b2006c4c.png)

**Comment by PhaserEditor2D** [comment]
Hi,

Do you mean this:

![image](https://user-images.githubusercontent.com/12187299/92748802-835d8200-f353-11ea-95a5-b36abcc08fd1.png)

**Comment by PhaserEditor2D** [comment]
Or do you mean to show the **Display Width** and **Display Height** in the **Transformation** section?

![image](https://user-images.githubusercontent.com/12187299/92749255-f5ce6200-f353-11ea-9025-172ceb65bbad.png)

**Comment by mikestonecodes** [comment]
yes showing the Display width and Display Height would fix it . but it would be nice to have total width and height of all selected items

**Comment by PhaserEditor2D** [comment]
How can we compute the total width and height?

For example, if two overlapping objects have a size of 100x100, does it show as total size 100x100 (remember both objects are overlapping)? Or 200x200, that is the total of the sizes?

**Comment by mikestonecodes** [comment]
Its the amount of pixels from top left of the object to the bottom right of whatever is drawn on the screen ( including rotation scale of each item ). This is how width and height of groups ( maybe containers also ) are calculated. that might be a bit complicated, just displaying the display width and display height for one object probably good.

**Comment by PhaserEditor2D** [comment]
Yes, we can start by showing the size of a single object because multiple objects have some problematic edge cases.
